coursedata/texts/es.yaml

```diff
@@ -20,9 +20,9 @@ ui:
     start_quiz: "Empezar quiz"
     end_quiz: "Terminar quiz"
     go_to_quiz: "Seleccionar quiz"
```

Comment:
Hi, sorry for the delay, maybe here instead of "quiz", which is an English word and may not be understood by people with no English background whatsoever, it would be better to translate it to "cuestionario".

Reply:
That's okay! Yeah, I was wondering whether to use a Spanish word or use the loanword from English. At this moment, quizes are not yet translated into Spanish, and the page uses the word "quiz" in a logo:

https://github.com/Felienne/hedy/blob/b033f40db6699e92d706dcb480cb482abba8be95/static/images/quiz_logo.jpg

This is what it looks like in Spanish:

<img width="538" alt="Screenshot of 'start quiz' page in Spanish, with the logo in English" src="https://user-images.githubusercontent.com/2294397/145975497-3de416d7-c250-48b8-8ccc-1da022ebb239.png">

I'll change it to "cuestionario" in a future PR anyway, but there's more work to do that just this!